src/staticanalysis/bot/static_analysis_bot/workflow.py

```diff
@@ -146,8 +149,16 @@ def run(self, revision):
             revision.analyze_patch()
 
         with stats.api.timer('runtime.mach'):
-            # Only run mach if revision has any C/C++ files
-            if revision.has_clang_files:
+            # Only run mach if revision has any C/C++ or Java files
+            if revision.has_clang_files or revision.has_infer_files:
```

Comment:
I guess `./mach configure` is needed for infer too ? :cry:

Reply:
Yes

Reply:
Unfortunately yes, we need to generate the grade solution.